tests/integration/test_cluster_discovery/config/config_observer.xml

```diff
@@ -3,7 +3,6 @@
     <remote_servers>
         <test_auto_cluster>
             <discovery>
-                <path>/clickhouse/discovery/test_auto_cluster</path>
```

Comment:
why removed?

Reply:
I moved it to separate file `config_discovery_path.xml`/`config_discovery_path_auxiliary_keeper.xml`.  There are two different paths for different tests (`/clickhouse/discovery/test_auto_cluster` for old one and `zookeeper2:/clickhouse/discovery/test_auto_cluster` for new one) , and I reused configs except discovery paths in both tests.